### PR TITLE
ME-13971-chapters-source

### PR DESCRIPTION
**Pull request by tsi**
Allow chapters to be configured at the source level.

### CODE REVIEW

File: src/plugins/vtt-thumbnails/index.js [hunk 1]
```diff
@@ -137,9 +137,9 @@ const VttThumbnailsPlugin = (function () {
       this.getChild('timeTooltip').updateTime(seekBarRect, seekBarPoint, time, () => {
         // Make sure the thumbnail doesn't exit the player
         if ((seekBarRect.width * seekBarPoint) < halfThumbnailWidth) {
-          this.el_.style.left = halfThumbnailWidth;
+          this.el_.style.left = `${halfThumbnailWidth}px`;
         } else if ((seekBarRect.width * seekBarPoint) + halfThumbnailWidth > width) {
-          this.el_.style.left = (seekBarRect.width * seekBarPoint) - halfThumbnailWidth;
+          this.el_.style.left = `${(seekBarRect.width * seekBarPoint) - halfThumbnailWidth}px`;
```
Comment (ehab-cl) [inline src/plugins/vtt-thumbnails/index.js:142]: why are these changes needed? like what are you trying to fix here?

**Review by ehab-cl — APPROVED** [review]
💪

File: src/plugins/vtt-thumbnails/index.js (hunk 1 above)
Comment (tsi) [reply src/plugins/vtt-thumbnails/index.js:142]: Depending on where you hovered, the seek-thumbs/chapters display wasn't getting positioned correctly.
This was happening mostly when mouse-entering from the sides directly over the progress-bar.
